cpu.cpp

```diff
@@ -857,8 +857,9 @@ inline bool CPU_QueryARMv7()
 #elif defined(_WIN32) && defined(_M_ARM64)
 	// Windows 10 ARM64 is only supported on Armv8a and above
 	return true;
-#endif
+#else

```

Comment:
The Linux and Android cases above are missing a `return` now in case the run-time conditions are not met.